### PR TITLE
feat(featured-items): add “featured items” component

### PR DESCRIPTION
**Pull request by yhuard**

### CODE REVIEW

File: framework/molecules/featured-items/featured-items--extended.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="featured-item featured-item__extended">
+  <div class="featured-item__image">
+    <img src="http://placehold.it/590x450" />
```
Comment (kalinchernev) [inline framework/molecules/featured-items/featured-items--extended.html]: Nice way to do it.

File: framework/molecules/featured-items/featured-items.config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  title: 'Featured items',
+  label: 'Featured items',
+  status: 'wip',
```
Comment (kalinchernev) [inline framework/molecules/featured-items/featured-items.config.js]: I think it's ready actually

File: framework/molecules/featured-items/featured-items.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<div class="featured-item">
+  <div class="featured-item__image">
+    <img src="http://placehold.it/590x450" />
+  </div>
+  <div class="featured-item__content">
+    <div class="meta meta--header">
```
Comment (kalinchernev) [inline framework/molecules/featured-items/featured-items.html]: Please include the `meta` atom as a peer dependency

File: framework/molecules/featured-items/featured-items--extended.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  </div>
+  <div class="featured-item__content">
+    <h3>Energy Union tour</h3>
+    <p>
+      Vice-President Šefcovic is visiting EU countries to present the opportunities that the Energy Union offers for Europe. The list of countries below are those he will visit over the next few months. Each completed visit comes with a presentation of the benefits of the Energy Union for that particular country. More visits will be added over time.
+    </p>
+    <a href="#" class="btn btn-ctn">Link Button</a>
```
Comment (kalinchernev) [inline framework/molecules/featured-items/featured-items--extended.html]: The buttons also seem to be a dependency.

File: framework/molecules/featured-items/featured-items.html (hunk 3 above)
Comment (yhuard) [inline framework/molecules/featured-items/featured-items.html]: Good catch! Thanks

**Comment by yhuard** [comment]
Fixed!

**Pull request merged**